tests/stream_reader_test.py

```diff
@@ -0,0 +1,56 @@
+import time
+
+import numpy as np
+import pytest
+
+from BusinessTampereTrafficMonitoring.stream_reader.stream_reader import StreamReader
+
+# Sample video from https://sample-videos.com
+SOURCE = "tests/samples_for_tests/test.mp4"
+
+
+@pytest.mark.parametrize("source", list("01345678:"))
```

Comment:
This parametrize doesn't seem necessary – if it works correctly with "0" as the invalid source, "1" will probably work too.

Reply:
To be fair this might not pass completely if there are any web cameras attached to the computer this is being ran on. I'll only put ':' there.